UnrealPlugin/README.md

```diff
@@ -20,7 +20,7 @@ The sample game these instructions were created with was called ThirdPersonMP, s
 ## Adding the plugin to the project
 When using the plugin a few things need to be taken care off.
 
-First, open your File Explorer and create a folder called “Plugins” in your games' root directory. Then drag the Playfab GSDK plugin file into the Plugins folder.
+First, open File Explorer and create a folder called “Plugins” in your games' root directory and in the Plugins folder, create a folder called "PlayfabGSDK". Then, drag all the Playfab GSDK Unreal plugin files from here into the Plugins/PlayfabGSDK folder.
```

Comment:
"plugin files from here": can we clarify where the 'here' is? It could be like the "UnrealPlugin" in this repo

Reply:
https://github.com/chris-gong/gsdk/commit/665206aa9237ae743b8ede09d8a0cf72021cf87a